test/flycheck-test.el

```diff
@@ -3547,6 +3543,17 @@ Why not:
    "language/processing/syntax_error/syntax_error.pde" 'processing-mode
    '(4 2 error "Syntax error, maybe a missing semicolon?"
        :checker processing)))
+(flycheck-ert-def-checker-test pug pug nil
+  (flycheck-ert-should-syntax-check
+   "language/pug/pug.pug" 'pug-mode
+   '(2 1 error "unexpected token \"indent\"" :checker pug))
+  (flycheck-ert-should-syntax-check
```

Comment:
Please split the test into multiple test cases.  There should only be one `-ert-should-syntax-check` call per `-ert-def-checker-test`.

The 3rd argument—which is `nil` currently—let's you change give a name to each checker test.